ports/luv/vcpkg.json

```
@@ -1,15 +1,14 @@
 {
   "name": "luv",
-  "version-semver": "1.44.2",
-  "port-version": 1,
+  "version-semver": "1.48.0-2",
   "description": "Bare libuv bindings for lua",
   "homepage": "https://github.com/luvit/luv",
   "license": "Apache-2.0",
   "supports": "!uwp",
   "dependencies": [
     {
       "name": "libuv",
-      "version>=": "1.44.2"
+      "version>=": "1.48.0-2"
```

```suggestion
      "version>=": "1.48.0"
```
`luv 1.48.0-2` depends on `libuv 1.48.0` https://github.com/luvit/luv/tree/1.48.0-2/deps

---

vcpkg currently only has libuv 1.46.0 so this constraint is broken